jenkins-builds/production.sh

```diff
@@ -7,21 +7,23 @@ scriptname=$(basename $0)
 
 usage() {
     echo "Usage: $0 [OPTIONS] <list-of-ebfiles>
-    -a,--arch     Architecture (gpu or mc)     (mandatory: Dom and Piz Daint only)
-    -f,--force    Force build of given package (optional: double quotes for a list)
+    -a,--arch     Architecture (gpu or mc)      (mandatory: Dom and Piz Daint only)
+    -f,--force    Force build of given package  (optional: double quotes for a list)
     -h,--help     Help message
-    -l,--list     Production list file         (mandatory: EasyBuild production list)
-    -p,--prefix   EasyBuild prefix folder      (mandatory: installation folder)
+    -l,--list     Production list file          (mandatory: EasyBuild production list)
+    -p,--prefix   EasyBuild prefix folder       (mandatory: installation folder)
+    -x,--noxalt  Disables XALT database update (optional)
```

Comment:
What do you think if we change it to the following?
`-x,--xalt [yes|no] update XALT database (optional, default is yes)`